src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -365,6 +367,10 @@ public int getFinalizerDelaySeconds() {
         return getInt("partitioner.finalizer.delay.seconds");
     }
 
+    public TimeZone getTimeZone(){
+        return Strings.isNullOrEmpty(getString("secor.parser.timezone")) ?  TimeZone.getTimeZone("UTC") : TimeZone.getTimeZone(getString("secor.parser.timezone"));
```

Comment:
This line is a bit too long, can you break it into two?  I think most of the coding style has 100 chars line width.